### PR TITLE
Assistant: Quick language update

### PR DESCRIPTION
**Pull request by jeffgolenski**
This is a simple one. 

Per a suggestion from @pyronaur, I'm updating the skip CTA on each step of the assistant from "Decide later" to "Not now"

> I’m someone who knows that I don’t want a certain feature, – I’m offered 2 options – “Set it up” or “Decide Later”. Neither of those two options align with what I want – which is not to enable it. Decide later makes it feel like I have to make “the right decision (to enable it)” later, which seems a bit forceful.

> “Not right now” is a slight shift in wording, it still communicates the fact that this can be turned on later, but also doesn’t imply that I will have to make that decision later.

**Before**
Screen Shot 2021-03-10 at 6 27 44 PM

**After**
Screen Shot 2021-03-10 at 6 27 58 PM

I know this is a massive PR and I should have broken it up into more than one ;-P

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Go to Jetpack > Dashboard > Recommendations.
* Start the recommendation flow.
* Check link text.

### CODE REVIEW

File: projects/plugins/jetpack/_inc/client/recommendations/prompts/feature-prompt/index.jsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ const FeaturePromptComponent = props => {
 						{ ctaText }
 					</Button>
 					<a href={ nextRoute } onClick={ onDecideLaterClick }>
-						{ __( 'Decide later', 'jetpack' ) }
+						{ __( 'Now now', 'jetpack' ) }
```
Comment (jeherve) [inline projects/plugins/jetpack/_inc/client/recommendations/prompts/feature-prompt/index.jsx:88]: Did you mean to write "Not now"?

```suggestion
						{ __( 'Not now', 'jetpack' ) }
```

**Review by jeherve — CHANGES_REQUESTED** [review]

File: projects/plugins/jetpack/_inc/client/recommendations/prompts/feature-prompt/index.jsx (hunk 1 above)
Comment (jeffgolenski) [reply projects/plugins/jetpack/_inc/client/recommendations/prompts/feature-prompt/index.jsx:88]: Ah yes. my ADHD was causing me some problems last night. :-( Thanks for the review jer.

Comment (jeherve) [reply projects/plugins/jetpack/_inc/client/recommendations/prompts/feature-prompt/index.jsx]: Thank you :)

In the PR description, you said:

> I know this is a massive PR and I should have broken it up into more than one ;-P

Is this PR ready for review now, or will you be adding more things in there? If it's ready for review, do you think you could edit the description and mark it as ready with the labels?

**Review by jeherve — APPROVED** [review]
This should be good to merge now (I've added a changelog entry and testing instructions, and tested). Merging.

**Pull request merged**